### DISCUSSION TITLE
KafkaRebalance refresh

### DISCUSSION THREAD

**Discussion by sabalde** [strimzi/strimzi-kafka-operator#8451] [General]
Hello,

What is the value to set on the annotation **strimzi.io/refresh** to force a KafkaRebalance refresh ? I can't find this info in the documentation.

Regards

**Comment by kyguy** [comment]
Hi @sabalde, the annotation is `strimzi.io/rebalance` and the value should be `refresh`. So to do a refresh you can annotate the resource like this:
```
kubectl annotate kafkarebalance <KAFKA_REBALANCE_RESOURCE> strimzi.io/rebalance=refresh -n <NAMESPACE>
```
Sorry this is not clear in the documentation, I will this gets updated

**Reply by sabalde** [reply]
Got it. Thank you @kyguy

**Reply by kyguy** [reply]
Updating docs here https://github.com/strimzi/strimzi-kafka-operator/pull/8453
